content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
 
 <TopBlock>
 
-This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality (Preview). Seeding allows you to consistently re-create the same data in your database and can be used to:
+This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality. Seeding allows you to consistently re-create the same data in your database and can be used to:
 
 - Populate your database with data that is required for your application to start - for example, a default language or a default currency.
-- Provide basic data for validating and using your application in a development environment. This is particulary useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
-
-Watch the following video for an introduction to the `db seed` command:
-
-<div class="videoWrapper">
-
-<iframe
-  width="560"
-  height="315"
-  src="https://www.youtube.com/embed/2LwTUIqjbPo"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
-
-</div>
+- Provide basic data for validating and using your application in a development environment. This is particularly useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
 
 </TopBlock>
 
-## Requirements for seeding with TypeScript or JavaScript
-
-Prisma's integrated seeding functionality expects the following:
-
-<!-- prettier-ignore-start -->
+## Configuring your seed script
 
-1. A `seed.ts` or `seed.js` file in the [same location as your `schema.prisma` file](../../concepts/components/prisma-schema#prisma-schema-file-location).
-2. **One** of the following inside the `seed.js`/`seed.ts` file:
+Prisma's integrated seeding functionality expects a command in the `"seed"` key in the `"prisma"` key of your `package.json` file. This points to a file which holds your seed script. The location of the file is optional, although we recommend placing it in the `/prisma` folder by convention.
```

Comment:
The `seed` key is any command, there is not necessarily a file in the project involved. I think we could use the content from https://github.com/prisma/docs/pull/2055 as a starting point.

Reply:
Hey @tomhoule , this is based off of that PR. Perhaps I wasn't clear in the above. I will re-write it.